JobConfig/primary/DIOtail.fcl

```diff
@@ -17,7 +17,7 @@ physics.producers.generate : {
 		        pdgId            :  11
 		        spectrumVariable : "totalEnergy"
 		        spectrumShape    : "tabulated"
-		        spectrumFileName : "ConditionsService/data/heeck_et_al_2021_aluminum.tbl"
+		        spectrumFileName : "ConditionsService/data/czarnecki_szafron_Al_2016_tail.tbl"
```

Comment:
I do not understand this change.   A related PR has added the "heeck" spectrum file, but now it is not used by anything?

Reply:
Yes, I thought I had changed this....let me check